readme.md

```diff
@@ -52,7 +52,7 @@ Contributions welcome. Add links through pull requests or create an issue to sta
 
 ### Communities
 
-- [MyDevSecOps](https://www.mydevsecops.io/) - _Snyk_ - A community that runs conferences, a blog, a podcast and a Slack workspace dedicated to DevSecOps.
```

Comment:
This link does need to be updated, but just a rename. MyDevSecOps is now DevSecCon, but it is still a community on discord.
https://www.devseccon.com/